examples/scenarioFlexiblePanel.py

```diff
@@ -0,0 +1,478 @@
+    attError.attRefInMsg.subscribeTo(inertial3DObj.attRefOutMsg)
+
+    return attError
+
+
+def setUpControl(scSim, extFTObject, attError, scGeometry):
+    IHubPntBc_B = np.array([[scGeometry.massHub / 12 * (scGeometry.lengthHub ** 2 + scGeometry.heightHub ** 2), 0.0, 0.0],
+                            [0.0, scGeometry.massHub / 12 * (scGeometry.widthHub ** 2 + scGeometry.heightHub ** 2), 0.0],
+                            [0.0, 0.0, scGeometry.massHub / 12 * (scGeometry.lengthHub ** 2 + scGeometry.widthHub ** 2)]])
+    IPanelPntSc_B = np.array(
+        [[scGeometry.massPanel / 12 * (scGeometry.lengthPanel ** 2 + scGeometry.thicknessPanel ** 2), 0.0, 0.0],
+         [0.0, scGeometry.massPanel / 12 * (scGeometry.widthPanel ** 2 + scGeometry.thicknessPanel ** 2), 0.0],
+         [0.0, 0.0, scGeometry.massPanel / 12 * (scGeometry.widthPanel ** 2 + scGeometry.lengthPanel ** 2)]])
+    r_ScB_B = [0.0, scGeometry.lengthHub / 2 + scGeometry.lengthPanel / 2,
+               scGeometry.heightHub / 2 - scGeometry.thicknessSubPanel / 2]
+    IHubPntB_B = IHubPntBc_B + IPanelPntSc_B - scGeometry.massPanel * np.array(rbk.v3Tilde(r_ScB_B)) @ np.array(
+        rbk.v3Tilde(r_ScB_B))
+
+    mrpControl = mrpFeedback.mrpFeedback()
+    mrpControl.ModelTag = "mrpFeedback"
+    scSim.AddModelToTask(scSim.fswTaskName, mrpControl)
+    decayTime = 50
+    xi = 0.9
+    mrpControl.P = 2 * np.max(IHubPntB_B) / decayTime
+    mrpControl.K = (mrpControl.P / xi) ** 2 / np.max(IHubPntB_B)
+
+    configData = messaging.VehicleConfigMsgPayload()
+    configData.IHubPntB_B = list(IHubPntB_B.flatten())
+    configDataMsg = messaging.VehicleConfigMsg()
+    configDataMsg.write(configData)
+    configDataMsg.this.disown()
+
+    mrpControl.guidInMsg.subscribeTo(attError.attGuidOutMsg)
+    mrpControl.vehConfigInMsg.subscribeTo(configDataMsg)
+    extFTObject.cmdTorqueInMsg.subscribeTo(mrpControl.cmdTorqueOutMsg)
+
+
+def setUpRecorders(scSim, scObject, panel, attError, scGeometry):
+    datLog = scObject.scStateOutMsg.recorder()
+    scSim.AddModelToTask(scSim.dynTaskName, datLog)
+    thetaData = []
+    for idx in range(scGeometry.numberOfSegments):
+        thetaData.append(panel.spinningBodyOutMsgs[2 * idx].recorder())
+        scSim.AddModelToTask(scSim.dynTaskName, thetaData[-1])
+        thetaData.append(panel.spinningBodyOutMsgs[2 * idx + 1].recorder())
+        scSim.AddModelToTask(scSim.dynTaskName, thetaData[-1])
+    attErrorLog = attError.attGuidOutMsg.recorder()
+    scSim.AddModelToTask(scSim.fswTaskName, attErrorLog)
+
+    return thetaData, attErrorLog
+
+
+def setUpVizard(scSim, scObject, panel, scGeometry):
+    scBodyList = [scObject]
+    for idx in range(scGeometry.numberOfSegments):
+        scBodyList.append([f"subPanel{idx + 1}", panel.spinningBodyConfigLogOutMsgs[2 * idx + 1]])
+    viz = vizSupport.enableUnityVisualization(scSim, scSim.dynTaskName, scBodyList
+                                              , saveFile=fileName
```

Comment:
This script is saving off a Vizard binary file by default.  This `safeFile` command should be commented out by default.

Reply:
Commented out that line.